GetIntoTeachingApi/Models/Crm/ICanGenerateUpfrontId.cs

```diff
@@ -0,0 +1,21 @@
+﻿using System;
+
+namespace GetIntoTeachingApi.Models.Crm
+{
+    public interface ICanGenerateUpfrontId
+    {
+        public Guid? Id { get; set; }
+        public bool ApiOnlyId { get; set; }
```

Comment:
This property reads funny to me; would it make more sense/link to the interface better as `HasUpfrontId`?